xlml/utils/xpk.py

```diff
@@ -60,14 +61,18 @@ def run_workload(
   """Run workload through xpk tool."""
 
   with tempfile.TemporaryDirectory() as tmpdir:
+    if accelerator_type == GpuVersion.XPK_H100.value:
```

Comment:
Is H100 the only supported GPU type?

Reply:
We might have A3+ too, which is also h100, but with a different keyword than `h100-80gb-8`.